### ISSUE TITLE
Strongkey

### ISSUE THREAD

**Issue by GoogleCodeExporter** [boddmg/nmodbus#26]
```
My software uses strong name keys.  When I try to reference the Modbus dll 
I get the error - Assembly generation failed - referenced 
assembly 'Modbus' does not have a strong name.  How would you recommend 
that I resolve this?  Thank you!


```

Original issue reported on code.google.com by `seminol...@knology.net` on 19 Mar 2008 at 6:17

**Comment by GoogleCodeExporter** [comment]
```
I've been meaning to strongly name the Modbus assembly, I'm posting a temporary
binary dist here 
http://groups.google.com/group/NModbus-discuss/web/NModbus_1.6.0.1.zip

Leaving the issue open as I have to think about whether or not to publicly 
release
the private key... 

Scott
```

Original comment by `sja...@gmail.com` on 21 Mar 2008 at 6:34

**Comment by GoogleCodeExporter** [comment]
```
Fixed with version 1.7
```

Original comment by `sja...@gmail.com` on 19 May 2008 at 2:00
- Changed state: **Fixed**
